gapic/configurable_snippetgen/configured_snippet.py

```diff
@@ -78,3 +78,26 @@ def filename(self) -> str:
         config_id = self.config.metadata.config_id
         sync_or_async = "sync" if self.is_sync else "async"
         return f"{module_name}_{self.api_version}_generated_{service_name}_{snake_case_rpc_name}_{config_id}_{sync_or_async}.py"
+
+    def _build_sample_function(self) -> None:
+        # TODO: add return type.
+        # TODO: add sample function parameters.
+        # TODO: add docstring.
+        # TODO: add sample function body.
+        pass
+
+    def _add_sample_function(self) -> None:
+        self._module = self._module.with_changes(
+            body=[self._sample_function_def])
+
+    def generate(self) -> None:
+        """Generates the snippet.
+
+        This is the main entrypoint of a ConfiguredSnippet instance, calling
+        other methods to update self._module.
+        """
+        self._build_sample_function()
+        self._add_sample_function()
+        # TODO: add imports.
+        # TODO: add region tag.
+        # TODO: add header.
```

Comment:
Please create issues in this repo for these Todo comments and the ones above

Reply:
Done.